CDS Maps/Individual Maps/FHIR-to-CDS/nNGM-FHIR-to-CDS-Operation.map

```diff
@@ -1,282 +1,306 @@
 /// version = 0.1
 /// title = "nNGM: Operation Therapie - FHIR to CTS"
 
+// Art der Resektion, Resektionstatus not mapped in to FHIR Map due to collate bug
```

Comment:
Is this still relevant? Let's solve this first 👍

Reply:
not relevant anymore